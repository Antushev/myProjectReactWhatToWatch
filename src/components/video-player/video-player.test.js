import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const isPlaying = false;
const posterImage = `picture.jpg`;
const previewVideo = `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`;

describe(`VideoPlayerComponent`, () => {
  it(`VideoPlayerComponentSnapshot`, () => {
    const tree = renderer
      .create(
          <VideoPlayer
            isPlaying={isPlaying}
            posterImage={posterImage}
            previewVideo={previewVideo}
            handleVideoPlayerMouseOver={() => {}}
            handleVideoPlayerMouseOut={() => {}}
          >
            <video />
          </ VideoPlayer>, {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
