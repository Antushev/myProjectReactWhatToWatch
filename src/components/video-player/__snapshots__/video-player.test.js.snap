// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerComponent VideoPlayerComponentSnapshot 1`] = `
<div
  className="small-movie-card__image"
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <video />
</div>
`;
